index.html

```diff
@@ -25,296 +24,172 @@
 </head>
 
 <body>
-    <header class="heading-element">
-        <h2 class="logo" id="logo2"><a href="#">Welcome</a></h2>
-        <nav>
-            <img src="./img/union.png" alt="menu icon" class="mobile-nav" id="hamburger">
-            <ul class="desk-nav" id="scripticon">
-               <li id="display"><img src="./img/close-btn.png " alt="close button"></li>
-                <li><a href="#portfolio-section">Portfolio</a></li>
-                <li><a href="#about-section">About</a></li>
-                <li><a href="#contact-section">Contact</a></li>
-            </ul>
-            </nav>
-            <div class="mob-menu collapse">
-                <i class="fa fa-times" aria-hidden="true"></i>
-                <ul class="menu-link">
-                   <li><a href="#portfolio-section" class="links">Portfolio</a></li>
-                   <li><a href="#about-section" class="links">About</a></li>
-                   <li><a href="#contact-section" class="links">Contact</a></li>
-               </ul>
-            </div>
-         
-    </header>
-    <section id="portfolio-section" class="home-page">
-        <div class="welcome-text">
-            <h1 class="intro">Hey there. I’m Theodore <br /><span>I’m a software<br />developer</span></h1>
-            <p class="info">
-                I can help you build a product , feature or website Look through some of
-                my work and experience! If you like what you see and have a project you
-                need coded, don’t hesitate to contact me.
-            </p>
-        </div>
-        <ul class="icon-list">
-            <li>
-                <a href="https://twitter.com/"><i class="fa fa-twitter" aria-hidden="true"></i></a>
-            </li>
-            <li>
-                <a href="https://www.linkedin.com/"><i class="fa fa-linkedin" aria-hidden="true"></i> </a>
-            </li>
-            <li>
-                <a href="https://medium.com/"><i class="fa fa-medium" aria-hidden="true"></i> </a>
-            </li>
-            <li>
-                <a href="https://github.com/"><i class="fa fa-github" aria-hidden="true"></i> </a>
-            </li>
-            <li>
-                <a href="https://angel.co/"><i class="fa fa-angellist" aria-hidden="true"></i> </a>
-            </li>
+  <div></div>
+  <header>
+    <div class="header">
+      <h3><a href="#">Welcome</a></h3>
+      <nav class="navbar_mobile_ver">
+        <img src="./images/Union.svg" onclick="mobileMenu()" alt="menu icon" />
+      </nav>
+    </div>
+    <div class="blur_bg">
+      <div class="mobile_menu">
+        <img src="./images/cross_icon.svg" class="cross_icon" onclick="closeMenu()" alt="cross icon">
+        <nav class="navbar_destop_version_alpha">
+          <a onclick="closeMenu()" href="#works">Portfolio</a>
+          <a onclick="closeMenu()" href="#about-me">About</a>
+          <a onclick="closeMenu()" href="#contact-us">contact</a>
+        </nav>
+      </div>
```

Comment:
## [OPTIONAL]

- [ ] For javascript best practice I think it should be better to put all the Javascript logic in a `js` file. I would kindly suggest you remove these `onclick` events in <a></a>. and use javascript for this implementation.